go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/google/uuid v1.1.2
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/grafana-plugin-model v0.0.0-20190930120109-1fc953a61fb4
-	github.com/grafana/grafana-plugin-sdk-go v0.81.0
+	github.com/grafana/grafana-plugin-sdk-go v0.82.0
```

Comment:
Seem comments in https://github.com/grafana/grafana-plugin-sdk-go/pull/249 (In particular the convert to proto, maybe do that and cut .83 with this?